modules/rostests/apitests/wininet/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 
-add_executable(wininet_apitest InternetOpen.c testlist.c)
+add_executable(wininet_apitest Download.c InternetOpen.c testlist.c)
```

Comment:
Files in apitests are generally named after a function rather than something generic.